src/main/resources/com/google/api/codegen/java/standalone_sample.snip

```diff
@@ -354,21 +356,17 @@
 @end
 
 @private processDefineView(view)
-  {@view.variableType} {@view.variableName} = {@variableField(view.reference)};
+  {@view.variableTypeName} {@view.variableName} = {@variableField(view.reference)};
 @end
 
 @private processPrintView(view)
-  @if not(view.args.size)
-    System.out.println("{@view.format}");
+  @if view.formattedString.args.isEmpty
+    System.out.println("{@view.formattedString.format}");
   @else
-    @if view.format == ""
-      System.out.println({@view.args.get(0).formattedName});
+    @if view.formattedString.format == "%s"
```

Comment:
we don't have `@elsif`, huh? Bummer.

Reply:
Unfortunately searching for both `elsif` and `elseif` gives nothing